mass_mailing_partner/README.rst

```diff
@@ -88,6 +88,8 @@ Contributors
     * Victor M.M. Torres
     * Manuel Calero
 
+* `Hibou Corp. <https://hibou.io>`_
```

Comment:
You should add this to readme/CONTRIBUTORS.rst instead

Reply:
Because the bot only regenerates it on merge (with extra commit). You can regenerate it locally and include the new file downloading OCA/maintainer-tools and running `oca-gen-addon-readme`, but OK if this has been added in the other file.